pkg/manager/member/tidb_member_manager.go

```diff
@@ -241,10 +243,87 @@ func (m *tidbMemberManager) syncTiDBStatefulSetForTidbCluster(tc *v1alpha1.TidbC
 			return err
 		}
 	}
+	// set random password
+	if tc.Spec.TiDB.Initializer != nil && tc.Spec.TiDB.Initializer.CreatePassword && tc.Status.TiDB.InitPasswordPhase == nil {
+		// sync password secret
+		secretName := controller.TiDBSecret(tc.Name)
+		_, err := m.deps.ServiceLister.Services(ns).Get(secretName)
+		isExistPasswordSecret := true
+		if err != nil {
+			if errors.IsNotFound(err) {
+				isExistPasswordSecret = false
+			} else {
+				return err
+			}
+		}
+		if !isExistPasswordSecret {
+			klog.Infof("Create random password for cluster %s/%s", tc.Namespace, tc.Name)
+			secret := m.buildRandomPasswordSecret(tc)
+			err := m.deps.TypedControl.Create(tc, secret)
+			if err != nil {
+				return err
+			}
+		}
+
+		tidbInitializerName := controller.TiDBInitializer(tc.Name)
+		_, err = m.deps.TiDBInitializerLister.TidbInitializers(ns).Get(tidbInitializerName)
+		isExistTidbInitializer := true
+		if err != nil {
+			if errors.IsNotFound(err) {
+				isExistTidbInitializer = false
+			} else {
+				return err
+			}
+		}
+		if !isExistTidbInitializer {
+			policy := corev1.PullIfNotPresent
+			tidbInitializer := &v1alpha1.TidbInitializer{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      fmt.Sprintf("%s-init", tc.Name),
+					Namespace: tc.Namespace,
+				},
+				Spec: v1alpha1.TidbInitializerSpec{
+					Image:           "tnir/mysqlclient",
```

Comment:
1. This image does not support ARM
2. The user may not be able to access this image and need to customize this.

Reply:
What about we just connect to TiDB directly without using initializer？
Refer to https://github.com/pingcap/tiup/pull/1700/files and https://github.com/pingcap/tidb-operator/blob/master/cmd/backup-manager/app/util/generic.go#L52

Reply:
And no need to consider if TLS is enabled for TiDB.